crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -292,7 +350,10 @@ impl IndexerStore for PgIndexerStore {
             .build_transaction()
             .read_only()
             .run(|conn| {
-                let mut builder = move_calls_dsl::move_calls.filter(move_calls_dsl::move_package.eq(package_name.clone())).into_boxed();
+                let mut builder = move_calls_dsl::move_calls.filter(move_calls_dsl::move_package.eq(package_name.clone()))
```

Comment:
the original `distinct()` does not work b/c of the PG query limit of DISTINCT ON
https://www.postgresql.org/docs/9.5/queries-select-lists.html#QUERIES-DISTINCT

> Note that the "first row" of a set is unpredictable unless the query is sorted on enough columns to guarantee a unique ordering of the rows arriving at the DISTINCT filter. (DISTINCT ON processing occurs after ORDER BY sorting.

Reply:
Also `group_by` needs to be done before `into_boxed` b/c Diesel's boxed query does not implement `GroupByDsl`

Reply:
> the original distinct() does not work b/c of the PG query limit of DISTINCT ON

I don't think this is the cause - see my other comment

Reply:
I'm checking why this was rejected...